frame/beefy/src/equivocation.rs

```diff
@@ -41,7 +41,8 @@ use frame_support::{
 };
 use frame_system::pallet_prelude::BlockNumberFor;
 use log::{error, info};
-use sp_consensus_beefy::{EquivocationProof, ValidatorSetId, KEY_TYPE};
+use sp_application_crypto::key_types::BEEFY as BEEFY_KEY_TYPE;
```

Comment:
NIT. You can remove the extra `sp-application-crypto` dependency by re-exporting the `BEEFY` key type in `sp_consensus_beefy` (as it has been done from [babe](https://github.com/paritytech/substrate/blob/cfe2432fdba3ff498b3c867c18bb4c25d0625a74/primitives/consensus/babe/src/lib.rs#L40) and [grandpa](https://github.com/paritytech/substrate/blob/cfe2432fdba3ff498b3c867c18bb4c25d0625a74/primitives/consensus/grandpa/src/lib.rs#L44) engines)